src/shared/components/MSKTabs/MSKTabs.tsx

```diff
@@ -141,10 +141,9 @@ export class MSKTabs extends React.Component<IMSKTabsProps> {
 
     private tabClickHandlers = MemoizedHandlerFactory(
         (
-            e: React.MouseEvent<any>,
+            _: React.MouseEvent<any>,
```

Comment:
Do we really need this change? This is going to shadow the imported lodash variable `import _ from 'lodash';`. It's not recommended to introduce shadowed variables even if they don't break anything (just like in this case).

Reply:
No, I'll revert it. I was trying to mark it as an unused variable, but I didn't think about lodash.

Reply:
yeah, typescript probably will complain if we don't specify the argument, even if we don't use it.  and "e" is fairly conventional for DOM events